packages/analytics/analytics-chart/src/utils/constants.ts

```diff
@@ -5,6 +5,8 @@ export const DECIMAL_DISPLAY = 2
 export const FONT_SIZE_SMALL = 10
 export const FONT_SIZE_REGULAR = 12
 
+export const CHART_REFRESH_INTERVAL_MS = 30 * 1000
```

Comment:
I think this should either go to `analytics-utilities` or remain in whatever host app is using `analytics-chart`